.kokoro/release.cfg

```diff
@@ -19,7 +19,7 @@ build_file: "google-cloud-ruby/.kokoro/trampoline_v2.sh"
 # Configure the docker image for kokoro-trampoline.
 env_vars: {
   key: "TRAMPOLINE_IMAGE"
-  value: "gcr.io/cloud-devrel-kokoro-resources/yoshi-ruby/release"
+  value: "us-central1-docker.pkg.dev/cloud-sdk-release-custom-pool/release-images/ruby-multi"
 }
 
 env_vars: {
```

Comment:
you'll also need to add SECRET_MANAGER_PROJECT_ID  env var set to cloud-sdk-release-custom-pool

Reply:
Yes, added.